spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :user do
+    sequence(:email) { |n| "user_#{n}@example.com"}
+    password {"password"}
+    password_confirmation {"password"}
```

Comment:
細かいですが式展開しない文字列はシングルクォートで囲んだ方が良いですね。
また波括弧の間はスペースで区切ると見やすいです。

```suggestion
    password { 'password' }
    password_confirmation { 'password' }
```